data/formats-data.js

```diff
@@ -5664,7 +5671,7 @@ exports.BattleFormatsData = {
 		tier: "NU",
 	},
 	binacle: {
-		randomBattleMoves: ["shellsmash", "liquidation", "stoneedge", "earthquake", "crosschop", "poisonjab", "xscissor", "rockslide"],
+		randomBattleMoves: ["shellsmash", "liquidation", "stoneedge", "crosschop", "switcheroo"],
```

Comment:
i dont think you need switcheroo - a one trick pony with SS is better

Reply:
ok, its bulk is enough